.ci/manywheel/build.sh

```diff
@@ -4,14 +4,17 @@ set -ex
 
 SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# Source the common build script for architecture-specific configurations (MKLDNN, ACL, etc.)
+source "${SCRIPTPATH}/../pytorch/build.sh" || true
```

Comment:
```suggestion
source "${SCRIPTPATH}/../pytorch/build.sh"
```

See https://github.com/pytorch/pytorch/issues/168003